app/src/main/java/com/jerboa/ui/components/home/HomeNavScreen.kt

```diff
@@ -221,134 +222,131 @@ fun BottomNavScreen(
         },
         modifier = Modifier.semantics { testTagsAsResourceId = true },
         content = {
-            Scaffold(
-                snackbarHost = { JerboaSnackbarHost(snackbarHostState) },
-                bottomBar = {
+            NavigationSuiteScaffold(
+                navigationSuiteItems = {
                     if (appSettings.showBottomNav && acc !== GuardAccount) {
-                        BottomAppBarAll(
+                        adaptiveNavigationBar(
                             selectedTab = selectedTab,
+                            userViewType = account.userViewType(),
                             unreadCounts = siteViewModel.unreadCount,
                             unreadAppCount = siteViewModel.unreadAppCount,
                             unreadReportCount = siteViewModel.unreadReportCount,
                             showTextDescriptionsInNavbar = appSettings.showTextDescriptionsInNavbar,
-                            userViewType = account.userViewType(),
-                            onSelect = onSelectTab,
+                            onSelectTab = onSelectTab,
                         )
                     }
                 },
-            ) { padding ->
-                NavHost(
-                    navController = bottomNavController,
-                    startDestination = NavTab.Home.name,
-                ) {
-                    composable(route = NavTab.Home.name) {
-                        HomeScreen(
-                            appState = appState,
-                            homeViewModel = homeViewModel,
-                            accountViewModel = accountViewModel,
-                            siteViewModel = siteViewModel,
-                            appSettingsViewModel = appSettingsViewModel,
-                            showVotingArrowsInListView = appSettings.showVotingArrowsInListView,
-                            useCustomTabs = appSettings.useCustomTabs,
-                            usePrivateTabs = appSettings.usePrivateTabs,
-                            drawerState = drawerState,
-                            blurNSFW = appSettings.blurNSFW.toEnum(),
-                            showPostLinkPreviews = appSettings.showPostLinkPreviews,
-                            markAsReadOnScroll = appSettings.markAsReadOnScroll,
-                            postActionBarMode = appSettings.postActionBarMode.toEnum(),
-                            swipeToActionPreset = appSettings.swipeToActionPreset.toEnum(),
-                            padding = padding,
-                        )
-                    }
+            ) {
+                Scaffold(
+                    snackbarHost = { JerboaSnackbarHost(snackbarHostState) },
+                    content = { padding ->
```

Comment:
It's strange to embed a scaffold within the adaptive scaffold, but its the only way to pass the snackbarhost for the error messages.